src/collection-preferences/__integ__/collection-preferences.test.ts

```diff
@@ -4,34 +4,46 @@ import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objec
 import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
 import createWrapper from '../../../lib/components/test-utils/selectors';
 
-class PageObject extends BasePageObject {}
+const setupTest = (testFn: (page: BasePageObject) => Promise<void>) => {
+  return useBrowser(async browser => {
+    const page = new BasePageObject(browser);
+    await browser.url('#/light/collection-preferences/simple');
+    await page.setWindowSize({ width: 1200, height: 1200 });
+    await testFn(page);
+  });
+};
 
 describe('Collection preferences', () => {
   test(
-    'renders 1 column',
-    useBrowser(async browser => {
-      await browser.url('#/light/collection-preferences/simple');
-      const page = new PageObject(browser);
-      await page.setWindowSize({ width: 1200, height: 1200 });
-
+    'renders no columns if there is only custom content',
+    setupTest(async page => {
       const wrapper = createWrapper().findCollectionPreferences('.cp-3');
       await page.waitForVisible(wrapper.findTriggerButton().toSelector());
       await page.click(wrapper.findTriggerButton().toSelector());
       await expect(page.isExisting(wrapper.findModal().toSelector())).resolves.toBe(true);
 
+      // The content is small enough so that it doesn't need column layout
       const columnLayout = wrapper.findModal().findContent().findColumnLayout();
-      await expect(page.isExisting(columnLayout.findColumn(1).toSelector())).resolves.toBe(true);
-      await expect(page.isExisting(columnLayout.findColumn(2).toSelector())).resolves.toBe(false);
+      await expect(page.isExisting(columnLayout.toSelector())).resolves.toBe(false);
```

Comment:
Is there other content we should be expecting here? The test asserts there is no column layout but it could also pass if there is no content at all that is rendered

Reply:
This is really just checking for the responsiveness behavior. Everything else is already covered in the unit tests. So we could check for other content to be there but it won't bring any additional value.

Reply:
But could the responsiveness result in all the content being hidden?

Reply:
fair point :)